compiler/rustc_attr_parsing/src/attributes/mod.rs

```diff
@@ -12,6 +12,15 @@
 //!
 //! Attributes should be added to [`ATTRIBUTE_GROUP_MAPPING`](crate::context::ATTRIBUTE_GROUP_MAPPING) to be parsed.
 
+use std::marker::PhantomData;
```

Comment:
this file is actually important: it defines the API various parsers for individual attributes will implement.